src/translations/fr/campaign.php

```diff
@@ -15,12 +15,12 @@
     'Plugin Name' => 'Newsletters',
     // -------------------------------------------------------------------------
     '“{name}” Campaigns' => '“{name}” Newsletter',
-    '“{name}” Mailing Lists' => '“{name}” Listes d’envoi',
-    '“Include original headers” must be enabled for each notification type.<br>See this <a href="{url}" target="_blank">article</a> for details on where and how to enable this.' => '',
+    '“{name}” Mailing Lists' => '“{name}” Listes de diffusion',
+    '“Include original headers” must be enabled for each notification type.<br>See this <a href="{url}" target="_blank">article</a> for details on where and how to enable this.' => 'L’option « Include original headers » doit être activée pour chaque type de notification. <br>Voir cet <a href=« {url} » target=« _blank »>article</a> pour plus de détails sur l’endroit et la manière d’activer cette option.',
     '(reply to {email})' => '(répondre à {email})',
     '{count} pending sendout(s) queued.' => '{count} envoi(s) en attente.',
     '{num, plural, =1{mailing list} other{mailing lists}}' => '{num, plural, =1{liste de diffusion} other{listes de diffusion}}',
-    'A campaign must be selected.' => '',
+    'A campaign must be selected.' => 'Une newsletter doit être sélectionnée',
```

Comment:
Is there a reason you’re leaving out full stops at the end of the French sentence? I’d like to keep it consistent here are everywhere else the English version ends in a full stop.

Reply:
Sorry for the lousy work. I've fixed the full-stops issue.

Reply:
Thank you!